### ISSUE TITLE
timeout-milliSeconds parameter in ADOMD query is actually seconds

### ISSUE THREAD

**Issue by alwortman** [Seddryck/NBi#551]
When running queries against a SSAS tabular model, the timeout-milliSeconds parameter appears to get translated into _seconds_ incorrectly.

System under test looks like:
```xml
<system-under-test>
  <execution>
    <query timeout-milliSeconds="30000" connection-string="@ProvDev"><![CDATA[
        EVALUATE
        FILTER(
          SELECTCOLUMNS (
            SUMMARIZECOLUMNS (
              'headcount_Hiring Manager'[Hiring Manager Name],
              "Measure Name", "Historical Total Open Requisitions",
              "Value", [Historical Total Open Requisitions]
            ),
            "Dimension Name", "'headcount_Hiring Manager'[Hiring Manager Name]",
            "Dimension Value", 'headcount_Hiring Manager'[Hiring Manager Name],
            "Measure Name", "Historical Total Open Requisitions",
            "Measure Value", [Historical Total Open Requisitions]
          ),
          [Dimension Value] <> ""
        )
        ]]></query>
  </execution>
</system-under-test>
```

I cloned the develop branch and ran the _ExecuteTestCases_ unit test. When the query executed, the timeout argument was in seconds instead of milliseconds.

![image](https://user-images.githubusercontent.com/13524879/67904146-c68f7780-fb2a-11e9-8354-ca9ba3b94c46.png)

The workaround is obviously to just lower the timeout to your desired threshold in seconds, but it is confusing as-is. 

Let me know if you need any additional info.

**Comment by Seddryck** [comment]
Thank you very much for the report.

I will fix that during the next days (or feel free to submit a PR) and it will be the final fix before v1.21 release.

**Comment by Seddryck** [comment]
Beta available at https://ci.appveyor.com/project/Seddryck/nbi/builds/28543510/artifacts or `1.22.0-beta0013` will also be included in v1.21
